sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosEndToEndOperationLatencyPolicyConfigBuilder.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.models.CosmosEndToEndOperationLatencyPolicyConfig;
+
+import java.time.Duration;
+
+/**
+ * Builder for CosmosEndToEndOperationLatencyPolicyConfig
+ */
+public class CosmosEndToEndOperationLatencyPolicyConfigBuilder {
+    private final boolean isEnabled;
+    private Duration endToEndOperationTimeout = Duration.ofSeconds(2); //defaults to two seconds
+
+    /**
+     * Create a CosmosEndToEndOperationLatencyPolicyConfigBuilder enabling the policy
+     */
+    public CosmosEndToEndOperationLatencyPolicyConfigBuilder() {
+        this(true);
+    }
+
+    /**
+     * Enable or disable the end to end timeout policy
+     * @param isEnabled Flag to toggle the end to end timeout policy
+     */
+    public CosmosEndToEndOperationLatencyPolicyConfigBuilder(boolean isEnabled) {
```

Comment:
why do we need this constructor?

Reply:
To enable/disable the Config. Hmm doesnt sound intutive actually!

Reply:
I think the ctor should only accept latency - that should be a required config - everything else should be optional for which setter is ok.

Reply:
So in that case user will be forced to provide a timeout even when they just want to disable the policy? @FabianMeiswinkel

Reply:
As discussed, timeout is forced in the builder constructor.